bucket/android-sms-backup-and-restore.json

```
@@ -0,0 +1,37 @@
+{
+    "version": "2.14.37",
+    "description": "Backup & Restore Android SMS on computer with ease.",
+    "homepage": "https://www.backuptrans.com/android-sms-backup-restore.html",
+    "license": "Unknow",
```

https://dictionary.cambridge.org/dictionary/english/unknown

```suggestion
    "license": "Unknown",
```